docs/MobileStepper.md

```diff
@@ -0,0 +1,73 @@
+# Mobile Stepper
+
+The `<pxb-mobile-stepper>` is used to minimally display progress when completing a workflow that requires multiple steps.  There are three types of progress indicators supported, `dots`, `text`, and `progress`. 
+
+<div style="text-align:center; margin-bottom:20px">
+    <img width="100%" style="max-width: 600px" alt="Mobile Stepper used " src="./images/dotStepper.png">
+</div>
+
+## Usage
+
+```typescript
+// app.module.ts
+import { MobileStepperModule } from '@pxblue/angular-components';
+...
+imports: [
+    MobileStepperModule
+],
+```
+
+```html
+// your-component.ts
+activeIndex = 0;
+steps = 4;
+```
```

Comment:
do we need to show these variable definitions? Would it be cleaner just to leave them out?

Reply:
For the code example to work, our end users will have to define `activeIndex` as a variable in their component.  It's their responsibility to keep track of which dot number is active.  We could leave out the `dots` variable though.

Reply:
it's not a big deal, I was just wondering if it made sense to simplify. We could hardcode numbers in the binding instead of using variables, but I wouldn't worry about it.